packages/roosterjs-editor-core/lib/coreApi/ensureTypeInContainer.ts

```diff
@@ -26,7 +27,12 @@ export const ensureTypeInContainer: EnsureTypeInContainer = (
     position: NodePosition,
     keyboardEvent?: KeyboardEvent
 ) => {
+    const table = findClosestElementAncestor(position.node, core.contentDiv, 'table');
+    if (table) {
+        position = new Position(table.querySelector('td,th'), PositionType.Begin);
```

Comment:
In some corner case, querySelector my return null? then position will be null

Reply:
Good point, fixing this.